lib/cfg-block.c

```diff
@@ -53,11 +53,11 @@ cfg_block_format_name(CfgBlockGenerator *s, gchar *buf, gsize buf_len)
 {
   CfgBlock *self = (CfgBlock *) s;
 
-  g_snprintf(buf, buf_len, "block %s %s (at %s:%d:%d)",
+  g_snprintf(buf, buf_len, "block %s %s() at %s:%d",
```

Comment:
() is left for separator purposes?

Reply:
In the documentation and log messages we use a convention that a paren is there at the end of the driver, e.g. unix-stream() instead of unix-stream.

This just follows that convention.